.whitesource

```diff
@@ -0,0 +1,3 @@
+{
+  "settingsInheritedFrom": "manageiq/whitesource-config@master"
```

Comment:
should this referenced file be included in this PR?

Reply:
It will be an entire separate repo that I will have to create, but I wanted to get this one file in first.